### ISSUE TITLE
implement sticky keys

### ISSUE THREAD

**Issue by kasimeka** [janw4ld/balatro-typist-mod#78]
instead of having to hold the jokers or consumables prefix keys, users should have a setting that allows them to toggle these keys with a single press. this is mainly at accessibility feature that helps with low dexterity and relieves the strain that comes from holding far away keys with your pinkys
